counterpartylib/test/parse_block_test.py

```diff
@@ -16,7 +16,7 @@ def test_parse_block(server_db):
     test_outputs = blocks.parse_block(server_db, DP['default_block_index'], 1420914478)
     outputs = ('74f02a316f377684fd1271cfe357c083f337feffc73afebc7a2c124ce95d30fe',
                '34d00501907f38d5fbe362d2cab991cfda7eb1bd3584b85b3e063ab3d045cd93',
-               '0faccf23ec33b79f92295d1b8de0a632ea323d0ea098e83b6e4a01fe44254dec',
+               'ca2fbf89ed6a7be04d432981818c4b77fab2f366fb3743e2950b5b592a19e48a',
```

Comment:
This change is because of a message hash miscalculation in the CIP 9 (subasset) tests.  I fixed that as part of this implementation which is why the message hashes are off.